static/css/around-map.css

```diff
@@ -1,8 +1,186 @@
+main {
```

Comment:
@newick: we are doing final reviews here. Would you feel like checking on this file?